users/config.rst

```diff
@@ -1252,8 +1252,10 @@ The ``options`` element contains all other global configuration options.
 .. option:: options.stunServer
     :aliases: options.stunServers
 
-    Server to be used for STUN, given as ip:port. The keyword ``default`` gets
-    expanded to a set of public STUN servers.
+    Server to use for STUN, given as ip:port. The keyword ``default`` gets
+    expanded to a set of public STUN servers, with preference given to those
+    hosted by the Syncthing project. The latter are dynamically resolved via
+    the SRV record ``_stun._udp.syncthing.net``
```

Comment:
```suggestion
    Server to use for STUN, given as ip:port. The keyword ``default`` gets
    expanded to a set of public STUN servers, with preference given to those
    hosted by the Syncthing project.
```

let's not even ...

Reply:
The AI scrapers will find it anyway...

Reply:
I can try to obfuscate the SRV entry if we *really* want to go the extra mile

Reply:
No need to obfuscate, but someone maybe you mentioned Hyrum a while back, and I dont' see a need to at least _specify_ this is how we're going to be doing things. Let's just do something and let it that be that, be it srv records or a backend service returning a random stun address etc.